### PR TITLE
README: add oc-cluster-up results to dashboard

### PR DESCRIPTION
**Pull request by miabbott**
This mostly adds the results of the `oc cluster up` test to the
dashboard.  I also cleared out the references to some of the
older/unsupported tests that we haven't maintained for a while.
(Maybe we should just nuke them?)

### CODE REVIEW

**Comment by mike-nguyen** [comment]
Is the oc cluster up test status missing for fedora-28-atomic-updates and fedora-rawhide?

File: README.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ You can click on the pass/fail badge to get taken to the log of the results.
...
 | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/cahc/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/cahc/latest/k8-cluster.log) | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/centosah/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/centosah/latest/k8-cluster.log) |
+| oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/cahc/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/cahc/latest/k8-cluster.log) | oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/centosah/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/centosah/latest/k8-cluster.log) |
```
Comment (mike-nguyen) [inline README.md]: s/k8-cluster.log/oc-cluster-up.log

File: README.md [hunk 2]
```diff
@@ -57,6 +58,7 @@ You can click on the pass/fail badge to get taken to the log of the results.
...
 | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fahc/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fahc/latest/k8-cluster.log) | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-27-atomic/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-27-atomic/latest/k8-cluster.log) |
+| oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fahc/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fahc/latest/k8-cluster.log) | oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-27-atomic/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-27-atomic/latest/k8-cluster.log) |
```
Comment (mike-nguyen) [inline README.md]: s/k8-cluster.log/oc-cluster-up.log

File: README.md [hunk 3]
```diff
@@ -68,6 +70,7 @@ You can click on the pass/fail badge to get taken to the log of the results.
...
 | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-27-atomic-testing/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-27-atomic-testing/latest/k8-cluster.log) | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-27-atomic-updates/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-27-atomic-updates/latest/k8-cluster.log) |
+| oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-27-atomic-testing/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-27-atomic-testing/latest/k8-cluster.log) | oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-27-atomic-updates/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-27-atomic-updates/latest/k8-cluster.log) |
```
Comment (mike-nguyen) [inline README.md]: s/k8-cluster.log/oc-cluster-up.log

File: README.md [hunk 4]
```diff
@@ -79,6 +82,7 @@ You can click on the pass/fail badge to get taken to the log of the results.
...
 | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-28-atomic/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-28-atomic/latest/k8-cluster.log) | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-28-atomic-testing/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-28-atomic-testing/latest/k8-cluster.log) |
+| oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-28-atomic/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-28-atomic/latest/k8-cluster.log) | oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-28-atomic-testing/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-28-atomic-testing/latest/k8-cluster.log) |
```
Comment (mike-nguyen) [inline README.md]: s/k8-cluster.log/oc-cluster-up.log

File: README.md [hunk 5]
```diff
@@ -90,6 +94,7 @@ You can click on the pass/fail badge to get taken to the log of the results.
...
 | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-28-atomic-updates/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-28-atomic-updates/latest/k8-cluster.log) | k8-cluster | [![k8-cluster log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-rawhide/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/k8-cluster/fedora-rawhide/latest/k8-cluster.log) |
+| oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-28-atomic-updates/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-28-atomic-updates/latest/k8-cluster.log) | oc-cluster-up | [![oc-cluster-up log](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-rawhide/latest/status.png)](https://s3.amazonaws.com/aos-ci/atomic-host-tests/oc-cluster-up/fedora-rawhide/latest/k8-cluster.log) |
```
Comment (mike-nguyen) [inline README.md]: s/k8-cluster.log/oc-cluster-up.log

**Comment by miabbott** [comment]
I didn't realize I could mess this up so much.  :disappointed:

**Comment by miabbott** [comment]
Another fixup :arrow_up:

**Comment by mike-nguyen** [comment]
LGTM

**Pull request merged**